test/functional/apps/discover/_data_view_editor.ts

```diff
@@ -14,10 +15,18 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const kibanaServer = getService('kibanaServer');
   const esArchiver = getService('esArchiver');
   const security = getService('security');
+  const config = getService('config');
   const PageObjects = getPageObjects(['common', 'discover', 'header', 'timePicker']);
+  const defaultIndexPatternString = config.get('esTestCluster.ccs')
+    ? 'ftr-remote:logstash-*'
+    : 'logstash-*';
   const defaultSettings = {
-    defaultIndex: 'logstash-*',
+    defaultIndex: defaultIndexPatternString,
   };
+  const localArchiveDirectory = 'test/functional/fixtures/kbn_archiver/discover';
+  const remoteArchiveDirectory = 'test/functional/fixtures/kbn_archiver/ccs/discover';
+  let esNode: EsArchiver;
+  let kbnDirectory: string;
```

Comment:
Is it possible to set `esNode` / `kbnDirectory` similar to `defaultIndexPatternString`?
I guess there is no reason to keep it mutable, right?

```
const esNode: EsArchiver = config.get('esTestCluster.ccs') ? getService('remoteEsArchiver' as 'esArchiver') : getService('esArchiver');
```

Reply:
Yeah, doing what @dmlemeshko is suggesting would be fine.

Reply:
My point was to keep consistency. If we do it for `defaultIndexPatternString`, we can do it for service and directory path.

Reply:
I think all services should be imported in the same place ideally:

```ts
const esNode = config.get('esTestCluster.ccs')
  ? getService('remoteEsArchiver' as 'esArchiver')
  : getService('esArchiver');
```

At the top of the file is ideal in my eyes.

Reply:
OK cool. I will fix and adjust.

Reply:
OK. Fixed.

Reply:
I don't see any changes